push_notifications/models.py

```diff
@@ -74,7 +78,6 @@ def send_message(self, message, **kwargs):
 					if reg_ids:
 						r = gcm_send_message(reg_ids, data, cloud_type, application_id=app_id, **kwargs)
 						response.append(r)
-
```

Comment:
No unnecessary changes in your commits please (keep style commits separate if you have to)